tpv104/fault.yaml

```diff
@@ -17,9 +17,11 @@
   function: |
       function f (x)
         x1 = math.min(18000.0, math.max(15000., math.abs(x["x"])))
-        z1 = math.min(18000.0, math.max(15000., math.abs(x["z"])))
+        z1 = math.min(10500.0, math.max(7500., math.abs(x["z"] + 7500)))
+
         box_x = 0.5*(1.0+math.tanh(3000.0/(x1-15000.0-3000.0) + 3000.0 / (x1-15000.0)))
-        box_z = 0.5*(1.0+math.tanh(3000.0/(z1-15000.0-3000.0) + 3000.0 / (z1-15000.0)))
```

Comment:
I think this was correct:
![image](https://github.com/SeisSol/Examples/assets/13536910/337f3091-2b27-49ce-b886-f88a8efe92a4)
https://strike.scec.org/cvws/download/SCEC_validation_slip_law.pdf

Reply:
![image](https://github.com/SeisSol/Examples/assets/24592027/d0887680-3751-40a4-89c9-8e308a147df6)
The boxcar function in z takes y-y0 and W/2 as arguments...

Reply:
I think you are right, I think y0 is not meant to be the hypocenter depth but 16.5km given the figure.

Reply:
with your version, the central portion of the fault would not be velocity weakening all the way to 15km depth.
![image](https://github.com/SeisSol/Examples/assets/13536910/74d980d0-e47f-4e6f-b332-5df38041d359)

Reply:
ok, after discussing I think there is no error in the benchmark description (that your code implements), and that the current implementation is also correct (actually more readable that the benchmark IMO).